### PR TITLE
fix 404

### PR DESCRIPTION
**Pull request by luffyke**
1. fix 404 for [从源代码安装](https://docs.gitea.io/zh-cn/hacking-on-gitea/zh-cn/installation/from-source) in [Hacking on Gitea](https://docs.gitea.io/zh-cn/hacking-on-gitea/) page.

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by metalmatze** [comment]
LGTM

**Comment by lunny** [comment]
@luffyke please also send a PR to release/v1.0

**Comment by luffyke** [comment]
@lunny sent

File: content/doc/advanced/hacking-on-gitea.zh-cn.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ menu:
 
 # Hacking on Gitea
 
-首先你需要一些运行环境，这和 [从源代码安装](zh-cn/installation/from-source) 相同，如果你还没有设置好，可以先阅读那个章节。
```
Comment (tboerger) [inline content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: This linking is generally simply wrong, please fix the link based on https://gohugo.io/extras/crossreferences/

Comment (luffyke) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: thanks, fixed.

Comment (lunny) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: @luffyke please send another PR to fix this.

Comment (luffyke) [reply content/doc/advanced/hacking-on-gitea.zh-cn.md:18]: @lunny have sent.

**Pull request merged**